### PR TITLE
Restarts reformat

### PR DESCRIPTION
**Pull request by robert-bryson**
Related to https://github.com/GSA/data.gov/issues/3326

### CODE REVIEW

File: .github/workflows/restart.yml [hunk 1]
```diff
@@ -1,94 +1,24 @@
...
-          LAST_RUN_BY: ${{ github.actor }}
-          RUN_ID: ${{ github.run_id }}
-        with:
-          filename: .github/restart_failure.md
-          assignees: ${{ github.actor }}
-          update_existing: true
+    uses: gsa/data.gov/.github/workflows/app-restart-template.yml@main
+    with:
+      environ: staging
+      app_names: "{\"include\":[{\"app\":\"inventory\"},{\"app\":\"inventory-proxy\"},]}"
```
Comment (btylerburton) [inline .github/workflows/restart.yml:15]: does this need to be an array?

Comment (btylerburton) [reply .github/workflows/restart.yml:15]: this run looks like it surfaces the error
https://github.com/GSA/inventory-app/actions/runs/4000944795

Comment (robert-bryson) [reply .github/workflows/restart.yml:15]: I wish. 

It's the horrible solution that was found for [catalog deploy](https://github.com/GSA/catalog.data.gov/blob/c0bee863a3594f1f1418c006d097739bde4ee14a/.github/workflows/publish.yml#L81) that I am just copying (note: the solution is clever but it reads horribly). I tried some yaml shenanigans, but it is a weird yaml in json in yaml thing that apparently needs to be in this format.

If you can find a better, more readable option please put it in. I hate things like this.

Comment (robert-bryson) [reply .github/workflows/restart.yml:15]: > this run looks like it surfaces the error https://github.com/GSA/inventory-app/actions/runs/4000944795

That run was from [this commit](https://github.com/GSA/inventory-app/pull/536/commits/aeb28ad2cf23091995cd18d30b939e0d2f4374d7). Note the commit message.

Comment (btylerburton) [reply .github/workflows/restart.yml:15]: ok. i gotcha. will the JSON break if there's a trailing comma? i see one here, but not in catalog-deploy.

Comment (btylerburton) [reply .github/workflows/restart.yml:15]: looks like it doesn't. voila!

**Review by btylerburton — APPROVED** [review]

File: .github/workflows/restart.yml (hunk 1 above)
Comment (robert-bryson) [reply .github/workflows/restart.yml:15]: It looks like it accepts it, but good catch cause there was a `prod` vs `production` issue.

Comment (robert-bryson) [reply .github/workflows/restart.yml:15]: For evidence: [successful run](https://github.com/GSA/inventory-app/actions/runs/4001082009)

**Pull request merged**